pkg/minishift/util/iso_path_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"path/filepath"
+	"testing"
+)
+
+func TestGetIsoPath(t *testing.T) {
+	testData := []struct {
+		provided string
+		expected string
```

Comment:
Nitpick: these variable `provided` should be changed to `isoURL`.

Reply:
@LalatenduMohanty why you think it should be `isoURL` since this is test data set and here we want what we provide and what expect from our tests. I am not convinced by this.

Reply:
@praveenkumar It is up to you. Personally `isoURLs` is more appropriate to me. But it shouldnot stop merging this PR.